<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>withdraw</class>
 <widget class="QDialog" name="withdraw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="verify_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="withdraw_label">
      <property name="text">
       <string>取款金额：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="wamount_edit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>verify_button</sender>
   <signal>clicked()</signal>
   <receiver>withdraw</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
